src/scss/_vars.scss

```diff
@@ -328,6 +334,8 @@ $snowflake-menu-padding: 1.25rem;
     --spacing-lg: 3rem;
     --spacing-xl: 4rem;
     --spacing-xxl: 5rem;
+    --select-icon-padding: 1.75rem;
```

Comment:
why have these in vars rather than directly in the select styles? seems a little further removed than necessary

Reply:
fair enough, it was late night decision